Makefile

```diff
@@ -70,7 +70,9 @@ docker:
 	docker build --tag="shopify/toxiproxy:git" .
 
 docker-release:
-	docker build --tag="shopify/toxiproxy:$(VERSION)" .
+	mkdir -p tmp
+	docker run -it --rm -v `pwd`/tmp:/go/bin -v `pwd`:/go/src/github.com/Shopify/toxiproxy golang:1.7 sh -c 'go build -ldflags="-X github.com/Shopify/toxiproxy.Version=$(cat /go/src/github.com/Shopify/toxiproxy/VERSION)" -o /go/bin/toxiproxy github.com/Shopify/toxiproxy/cmd && go build -o /go/bin/toxiproxy-cli github.com/Shopify/toxiproxy/cli'
```

Comment:
I think that would tie `docker-release` target in the makefile to a different target in the makefile that builds the linux binary. Specifically, make docker-release is only done when we want to release / push a new docker image.    

although one could argue that `make release` does `all` and `docker-release`, at which point the linux binaries should be present.